### ISSUE TITLE
[SUGGESTION]: add X attack and X click in the pokemart of both DLC maps of Galar

### ISSUE THREAD

**Issue by milotic** [pokeclicker/pokeclicker#5512]
### Current Behavior

i need buy both items to fight with peony and the pokemart only sell balls

### Improved Behavior

quick access to a basic item in pokeclicker

### Reason for change

both maps are endgame zones, but the pokemarts dont have items

### Detailed Description

better stores

**Comment by Qwertypop04** [comment]
Why can't you use the pokemart shortcut?

**Comment by milotic** [comment]
Oops, I totally forgot that button exists because with so many changes in the interface it is no longer at the end of the modules. Currently on the main screen the shortcut is not visible and if I scrolled at the end now there was the Underground, the store is a small access between the eggs and the farm, which I looked to see if it still existed because you just mentioned it. I think my muscle memory took that the shortcut no longer existed to make room for the new modules.

![imagen](https://github.com/user-attachments/assets/65be940a-4658-4890-a82a-fe43f8aa3897)
